frontend/packages/kubevirt-plugin/integration-tests/tests/utils/mocks.ts

```diff
@@ -252,52 +296,19 @@ export function getVMManifest(
             'vm.kubevirt.io/name': metadata.name,
           },
         },
-        spec: {
-          domain: {
-            cpu: {
-              cores: 1,
-              sockets: 1,
-              threads: 1,
-            },
-            devices: {
-              disks,
-              inputs: [
-                {
-                  bus: 'virtio',
-                  name: 'tablet',
-                  type: 'tablet',
-                },
-              ],
-              interfaces: [
-                {
-                  bootOrder: 2,
-                  masquerade: {},
-                  name: 'nic0',
-                  model: 'virtio',
-                },
-              ],
-              rng: {},
-            },
-            resources: {
-              requests: {
-                memory: '1073741824',
-              },
-            },
-          },
-          evictionStrategy: 'LiveMigrate',
-          terminationGracePeriodSeconds: 0,
-          networks: [
-            {
-              name: 'nic0',
-              pod: {},
-            },
-          ],
-          volumes,
-        },
+        spec,
       },
     },
   };
-  return vmResource;
+
+  const vmiResource = {
+    apiVersion: 'kubevirt.io/v1alpha3',
+    kind: 'VirtualMachineInstance',
+    metadata,
+    spec,
+  };
+
+  return kind === 'VirtualMachineInstance' ? vmiResource : vmResource;
```

Comment:
can we rather have two functions getVMManifest and getVMIManifest which would use common settings of them?

Reply:
done :+1: